server/__init__.py

```diff
@@ -20,10 +20,31 @@
 from google.cloud import storage
 from werkzeug.utils import import_string
 
+from opencensus.ext.flask.flask_middleware import FlaskMiddleware
+from opencensus.ext.stackdriver.trace_exporter import StackdriverExporter
+from opencensus.trace.propagation import google_cloud_format
+from opencensus.trace.samplers import AlwaysOnSampler
+from opencensus.ext.stackdriver import trace_exporter as stackdriver_exporter
+import opencensus.trace.tracer
+
+propagator = google_cloud_format.GoogleCloudFormatPropagator()
+
+
+def createMiddleWare(app, exporter):
+    # Configure a flask middleware that listens for each request and applies automatic tracing.
```

Comment:
nit: line width

Reply:
Done

Reply:
Not very sure if this will work, will see..